views/partials/footer.tmpl

```diff
@@ -1,2 +1,8 @@
 {{define "footer"}}
+	<footer class="container outer footer">
+        	<div class="navbar-left"><a href="#">Disclaimer</a> | <a href="#">MyRadio</a> | <a href="#">Contact</a></div>
+        	<div class="navbar-right">
+				<p>Designed by <a href="#">Computing</a></p>
```

Comment:
Should probably be "URY Computing Team" with a mailto link like the current site. Probably a copyright notice as well, because whynot